registry.json

```diff
@@ -1363,6 +1363,12 @@
         "path": "m/44'/136'/0'/0/0",
         "xpub": "xpub",
         "xprv": "xprv"
+      },
+      {
+        "name": "exchange",
+        "path": "m/44'/136'/0'/0/0",
+        "xpub": "xpub",
+        "xprv": "xprv"
       }
```

Comment:
We do not add derivations for such purposes. Instead, could you consider adding new `Prefix` variant like for Filecoin delegated address:
https://github.com/trustwallet/wallet-core/blob/c0cf03fe36c0b3116b7913bcb199cf3017f966b1/src/CoinEntry.h#L34-L37
FFI:
https://github.com/trustwallet/wallet-core/blob/c0cf03fe36c0b3116b7913bcb199cf3017f966b1/include/TrustWalletCore/TWAnyAddress.h#L117-L123
Usage:
https://github.com/trustwallet/wallet-core/blob/c0cf03fe36c0b3116b7913bcb199cf3017f966b1/src/Filecoin/Entry.cpp#L19-L24